prdoc/pr_4831.prdoc

```diff
@@ -0,0 +1,25 @@
+title: "treasury pallet: - remove unused config parameters"
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      Remove unused config parameters `ApproveOrigin` and `OnSlash` from the treasury pallet.
+      Add `OnSlash` config parameter to the bounties and tips pallets. 
+
+crates:
+  - name: pallet-treasury
+    bump: major
+  - name: pallet-bounties
+    bump: major
+  - name: pallet-tips
+    bump: major
```

Comment:
Updating to this version of tips and bounties with the additional associated type in their `Config` trait will make runtimes fail to compile without changes, so I think these are correct, despite what the semver check says.